src/js/libgeo/GeoBasics.js

```diff
@@ -118,6 +119,12 @@ function segmentDefault(el) {
     el.clip = General.string("end");
 }
 
+
+function polygonDefault(el) {
+    el.filled = (el.filled !== "undefined" ? General.bool(el.filled) : General.bool(true));
```

Comment:
You mean `el.filled !== undefined` without the quotes. And you should use `General.bool(!!el.filled)` or some such, since we don't know whether `el.filled` really is a boolean value.